lib/tasks/sample_data/enterprise_factory.rb

```diff
@@ -0,0 +1,93 @@
+require "tasks/sample_data/addressing"
+require "tasks/sample_data/logging"
+
+class EnterpriseFactory
+  include Logging
+  include Addressing
+
+  def create_samples(users)
+    log "Creating enterprises:"
+    enterprise_data(users).map do |data|
+      name = data[:name]
+      log "- #{name}"
+      Enterprise.create_with(data).find_or_create_by_name!(name)
+    end
+  end
+
+  private
+
+  # rubocop:disable Metrics/MethodLength
+  def enterprise_data(users)
+    [
+      {
+        name: "Penny's Profile",
+        owner: users["Penny Profile"],
+        is_primary_producer: false,
+        sells: "none",
+        address: address("25 Myrtle Street, Bayswater, 3153"),
+        long_description: <<DESC
```

Comment:
[strip_heredoc](https://apidock.com/rails/v3.2.13/String/strip_heredoc) is perfect for this.

Reply:
don't worry, it was just a suggestion to make you aware of that method if you weren't